apps/web/components/landing/mdx-content.tsx

```diff
@@ -0,0 +1,14 @@
+"use client"; // This is required!
```

Comment:
why?

Reply:
There is a useState used to display the serialized data.